Android/RenderScriptV8/samples/NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <RestoreNoCache>true</RestoreNoCache>
```

Comment:
This will just stop the restore from looking in the cache - and use the nuget from the output - but it will still write the package to the cache. To prevent this, you may have to redirect the restore somewhere else:

```
<RestorePackagesPath>$(MSBuildThisFileDirectory)..\externals\packages</RestorePackagesPath>
```